Assets/script/index.js

```diff
@@ -21,3 +59,20 @@ closeMenuBtn.addEventListener("click", () => {
   headerTag.style.backgroundColor = "#ffffff";
   menuContainer.style.margin = "1rem 2rem";
 });
+
+//dynamic speakers
+
+for (let i = 0; i < featuredSpeakers.length; i++) {
+  document.querySelector(".featured-speakers .speakers-card").innerHTML += `
+    <div class="card">
+    <img src="${featuredSpeakers[i].imageSource}" alt="${featuredSpeakers[i].name}" />
+    <div class="description">
+      <h3>${featuredSpeakers[i].name}</h3>
+      <p class="role">${featuredSpeakers[i].role}</p>
+      <img src="./Assets/image/icon/speaker_decoration.png" alt="underline" class="speaker-decoration" />
+      <p class="small">
+        ${featuredSpeakers[i].talk}
+      </p>
+    </div>
+    `;
+}
```

Comment:
@Kingstalux Kindly this more button is optional one it is stated in the requirement page.
<img width="1217" alt="Screen Shot 2022-05-18 at 12 49 33 PM" src="https://user-images.githubusercontent.com/56404835/169011255-fd524f19-8244-467d-823c-b6959e6c95d2.png">

Reply:
Hello @darikmohammed, since the functionality of the `more` button is optional, just stick to the original design and display 2 speakers in the mobile view with the button but don't add the functionality 👍

Reply:
@Kingstalux Thank you.